packages/@aws-cdk/aws-appmesh/lib/client-policy.ts

```diff
@@ -1,113 +1,26 @@
-import * as acmpca from '@aws-cdk/aws-acmpca';
-import { CfnVirtualNode } from './appmesh.generated';
-
-// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
-// eslint-disable-next-line no-duplicate-imports, import/order
-import { Construct } from '@aws-cdk/core';
-
-enum CertificateType {
-  ACMPCA = 'acm',
-  FILE = 'file',
-}
+import { TlsValidation } from './tls-validation';
 
```

Comment:
This file should be called `tls-client-policy.ts` (same way it's `tsl-validation.ts`, not `validation.ts`).

Reply:
Changed the name to `tls-client-policy.ts`